server/main.go

```diff
@@ -26,14 +28,16 @@ var cfg = flag.String("config", "config.yaml", "path to the config file")
 func main() {
 
 	flag.Parse()
+	root := flag.Arg(0)
```

Comment:
I think we should have a default here. Probably some directory in the working directory, such as: `./.tracetest` or so.